style.css

```diff
@@ -0,0 +1,73 @@
+section {
+    margin:auto;    
+    width: 625px;
+    background-color: lightgray;
+    color: white;
+}
+
+article {
+    display: inline-block;
+    width: 625px;
```

Comment:
No need to set the width here because it is the same as the parents

Reply:
I've made all the changes, but when i remove the article width, the background image disappears from the right part of the article, and i cant figure out why.  Any tips?

Reply:
Nvm i had to change inline-block to block in the article. Thank you for the thorough and kind review :)